keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -954,6 +953,7 @@ Major milestones might include:
 - 2021.01.12: KEP proposed for review, including motivation, proposal, risks,
 test plan and graduation criteria.
 - 2022.09.22: Graduate to Beta in v1.26.
+- 2024.10.10: Graduate to GA in v1.32.
```

Comment:
@kerthcet we didn't make it to GA in 1.32, right? If so, let's conclude it in 1.33.

Reply:
yes, because of we're approaching the deadline at that moment, I will target this for 1.33.